lib/please_cli/please_cli/local_config.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+import toml
+
+import cli_common.cli
+import cli_common.log
+
+log = cli_common.log.get_logger(__name__)
+
+
+class LocalConfig(object):
+    '''
+    Config for developers, in their home folder
+    '''
+    def __init__(self):
+        self.path = os.path.expanduser('~/.pleaserc')
```

Comment:
Lets use [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) for the location of the user configuration.

We should read from `XDG_CONFIG_HOME` system variable and default to `~/.config`. And for future lets us also create a folder where we put please configuration. User configuration should then be in `~/.config/please/config.toml`

Reply:
Ok :+1: